tox.ini

```diff
@@ -9,6 +9,7 @@ env_list = format, lint, unit
 [vars]
 src_path = {tox_root}/src
 tests_path = {tox_root}/tests
+test_charm_libs_path = {[vars]tests_path}/integration/app-charm/lib/
```

Comment:
We shouldn't touch the libs in the test charm.

Reply:
Should we release it as data-test-app? :-)